backend/controller/ml_model/templates/output.sql

```diff
@@ -0,0 +1,95 @@
+CREATE OR REPLACE TABLE `{{project_id}}.{{model_dataset}}.output` AS (
+  WITH prior_table AS (
+    SELECT SPLIT(MAX(table_id), 'events_')[OFFSET(1)] AS suffix
+    FROM `{{project_id}}.{{ga4_dataset}}.__TABLES_SUMMARY__`
+    WHERE table_id LIKE 'events_%'
+    AND table_id NOT IN (
+      SELECT MAX(table_id) FROM `{{project_id}}.{{ga4_dataset}}.__TABLES_SUMMARY__`)
+  ),
+  events AS (
+    SELECT DISTINCT
+      {{unique_id}},
+      event_name AS name
+      event_params AS params
+    FROM `{{project_id}}.{{ga4_dataset}}.events_*`
+    WHERE _TABLE_SUFFIX = (SELECT suffix FROM prior_table)
+  ),
+  users_with_score AS (
+    SELECT DISTINCT
+      {{unique_id}}
+    FROM events, UNNEST(params) AS params
+    WHERE name = 'prop_score'
+    AND params.value.string_value = 'Predicted_Value'
+  ),
+  users_without_score AS (
+    SELECT DISTINCT
+      {{unique_id}}
+    FROM events
+    WHERE {{unique_id}} NOT IN (
+      SELECT {{unique_id}} FROM users_with_score)
+  ),
+  {% if label.is_score %}
```

Comment:
is the value of label mutually exclusive, ie, can label.is_score and label.is_conversion both be true?

Reply:
Yes, they can both be true. But is_conversion can only be true if is_score is also true.